docs/content/posts/2023-11-13-release-1.17.2.md

```diff
@@ -0,0 +1,204 @@
+---
+title:  "Apache Flink 1.17.2 Release Announcement"
+date: "2023-11-13T00:00:00.000Z"
+aliases:
+- /news/2023/11/13/release-1.17.2.html
+authors:
+- myasuka:
+  name: "Yun Tang"
+  twitter: "yun_tang_"
+---
+ 
+The Apache Flink Community is pleased to announce the second bug fix release of the Flink 1.17 series.
+ 
+This release includes 104 bug fixes, vulnerability fixes, and minor improvements for Flink 1.17.
```

Comment:
I see [84 issues](https://issues.apache.org/jira/browse/FLINK-33474?jql=project%20%3D%20FLINK%20AND%20fixVersion%20%3D%201.17.2)

Reply:
Hmm, thanks for pointing out this issue.